### PR TITLE
Lower support for dev tools for backwards compatibility

### PR DESCRIPTION
**Pull request by mariuskurgonasativy**

### CODE REVIEW

**Comment by mariuskurgonasativy** [comment]
I would wait for merging this until you want to make source code private.

**Review by simontjw — APPROVED** [review]

File: GetivySDK.podspec [hunk 1]
```diff
@@ -16,8 +16,10 @@ s.author = { "Simon Wimmer" => "simon.wimmer@getivy.de" }
 
 s.homepage = "https://github.com/getivy/ios-sdk"
 
-s.source = { :git => "https://github.com/getivy/ios-sdk.git", 
-             :tag => "#{s.version}" }
+# s.source = { :git => "https://github.com/getivy/ios-sdk.git",
```
Comment (erkki-lukk) [inline GetivySDK.podspec:19]: Is this the same as on Android to use either local branch of repo dependency? If so let's add a comment please

File: Source/GetivySDKImpl+GetivySDKContract.swift [hunk 2]
```diff
@@ -69,7 +69,8 @@ extension GetivySDKImpl: GetivySDKContract {
                         // TODO: Double check this as checkout session does not return a market
                         market: "DE",
                         type: type,
-                        locale: result.locale
+                        locale: result.locale,
+                        currencry: result.price.currency
```
Comment (erkki-lukk) [inline Source/GetivySDKImpl+GetivySDKContract.swift:73]:
```suggestion
                        currency: result.price.currency
```

File: GetivySDK.podspec (hunk 1 above)
Comment (mariuskurgonasativy) [reply GetivySDK.podspec]: no, its controlled in the app level - removed comment

File: Source/GetivySDKImpl+GetivySDKContract.swift (hunk 2 above)
Comment (mariuskurgonasativy) [reply Source/GetivySDKImpl+GetivySDKContract.swift]: Addressed that in the commit

**Review by erkki-lukk — APPROVED** [review]

**Pull request merged**